### DISCUSSION TITLE
5.15编译这么费时间的吗

### DISCUSSION THREAD

**Discussion by DestoryG** [coolsnowwolf/lede#11337] [Q&A / 快速问答]
编译时长超过了6个小时以至于最后被Github强制终止了
![image](https://github.com/coolsnowwolf/lede/assets/136814177/5e550c51-450f-4992-89ee-371ac82b71f6)

**Comment by Cathgao** [comment]
微软给的就一台双核破机子，编译nodejs之类乱七八糟的东西当然跑不完

**Comment by WYC-2020** [comment]
别编译那个 node 这玩意 两个多小时，不编译这个 的话 adg 还有几个 库 也不能选

**Reply by DestoryG** [reply]
要不是网易云解灰要那个谁愿意编译那么老大个node😭

**Reply by hiddenblue** [reply]
我记得里面有个go版本的可以切换，go编译起来很快，就是联网下载。nodejs那个v8要编译老半天🤡😭

**Comment by szdosar** [comment]
你试试timeout-minutes这个参数，不知道现在是否被拿掉了。欢迎回来反馈。

`name: 示例工作流程

on: [push]

jobs:
  build:
    runs-on: ubuntu-latest
    timeout-minutes: 420 # 设置超时时间为7小时

    steps:
    - uses: actions/checkout@v2
    - name: 运行长时间任务
      run: |
        # 长时间运行的命令
`
